packages/wdio-cli/src/types.ts

```diff
@@ -90,9 +90,12 @@ export interface InstallCommandArguments {
     name: string
 }
 
+export type Framework = 'mocha' | 'jasmine' | 'cucumber'
```

Comment:
This should be moved into `@wdio/types` and applied in other areas where we type `framework`